pkg/core/native/native_neo.go

```diff
@@ -893,7 +908,7 @@ func (n *NEO) getCandidates(d *dao.Simple, sortByKey bool) ([]keyWithVotes, erro
 // GetCandidates returns current registered validators list with keys
 // and votes.
 func (n *NEO) GetCandidates(d *dao.Simple) ([]state.Validator, error) {
-	kvs, err := n.getCandidates(d, true)
+	kvs, err := n.getCandidates(d, true, -1)
```

Comment:
Not limited? This one is accessible via RPC.

Reply:
Fixed.